yuniql-core/ConfigurationService.cs

```diff
@@ -151,12 +151,13 @@ public void Validate()
         }
 
         ///<inheritdoc/>
-        public string PrintAsJson(bool redactSensitiveText = true)
+        public string PrintAsJson()
         {
             var _configuration = GetConfiguration();
```

Comment:
As convention, we dont use underscore for local variables. We only use underscore `_` for local private class level variables. Please name is as `configuration` unless it has name conflicts.

Reply:
thanks for the feedback! :) fixed with latest commit